quasar.config.js

```diff
@@ -108,9 +108,7 @@ module.exports = configure((ctx) => ({
     env: {
       TEMPLATE_LIBRARY_BASE_URL: process.env.TEMPLATE_LIBRARY_BASE_URL || configuration?.templateLibrary || '',
       VERSION: version,
-      AUTHENTICATION: configuration?.authentication?.OIDC
-        ? JSON.stringify(configuration.authentication.OIDC)
-        : '',
+      IFBACKEND: configuration?.backend ? Object.prototype.hasOwnProperty.call(configuration?.backend, 'url') && Object.prototype.hasOwnProperty.call(configuration?.backend, 'appId') : false,
```

Comment:
IFBACKEND ???
may be HAS_BACKEND or something better

Reply:
done